.gitmodules

```diff
@@ -1,3 +1,3 @@
-[submodule "cosmos-sdk"]
-	path = cosmos-sdk
-	url = https://github.com/cosmos/cosmos-sdk
+[submodule "wasmd"]
+	path = wasmd
+	url = https://github.com/cosmwasm/wasmd
```

Comment:
Does this contain all comos types? I guess we need to prepare for a multi source setup somehow.

Reply:
Can we rename this folder to `wasmd-0.16` and check out at v0.16.0? v0.17.0 contains some .proto changes that are compatible and for wasmd-0.18 we'll need a second source if we want to support the older networks.

Reply:
This is already checked out at v0.16, but I can rename it to prepare for the multiple sources.

Reply:
And yes, it contains all the cosmos types. I thought it was neat because we keep the versions in sync, but maybe that won't be sufficient once we have multiple wasmd versions.

Reply:
It's a good and important start. One thing I would try next is merge cosmos-sdk-0.43 into the output and see what happens. Ideally we only see compatible overrides and additions.

Reply:
OK, I'll merge this then and add that in another PR.